documentation/todo-app/src/components/todoMain.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import TodoItems from './todoItems';
+import './todo_main.css'
```

Comment:
Please be careful with your semicolon use. You can choose to either use semicolons or not use semicolons, but whichever way you choose, you must be consistent.